deep_qa/data/datasets/dataset.py

```diff
@@ -6,13 +6,24 @@
 import numpy
 import tqdm
 
-from ..common.util import add_noise_to_dict_values
-from .data_indexer import DataIndexer
-from .instances.instance import Instance, TextInstance, IndexedInstance
+from deep_qa.common.util import add_noise_to_dict_values
```

Comment:
Sorry, I let Pycharm do some refactoring of a variable name for me and it destroyed so many things, I thought i'd got them all but apparently not.